authentication/account/repository/identity_blackbox_test.go

```diff
@@ -79,6 +80,25 @@ func (s *IdentityRepositoryTestSuite) TestLoad() {
 		require.NoError(t, err, "Could not load identity")
 		assert.Equal(t, identity.Identity().Username, result.Username)
 	})
+
+	s.T().Run("ok load deleted identity by dynamic condition", func(t *testing.T) {
+		// given
+		g := s.NewTestGraph(t)
+		identity1 := g.CreateIdentity()
+
+		// when soft delete identity
+		err := s.Application.Identities().Delete(s.Ctx, identity1.ID())
+		// then
+		require.Nil(t, err)
+
+		identity, err := s.Application.Identities().Load(s.Ctx, identity1.ID(), func(db *gorm.DB) *gorm.DB {
```

Comment:
this func could be extracted to something like `includeSoftDeletes` for more clarity in the code